Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -1827,13 +1825,46 @@ public void setIdentity(@NonNull String userId, @Nullable BranchReferralInitList
         }
     }
 
+    /**
+     * Gets all available cross platform ids.
+     *
+     * @param callback An instance of {@link io.branch.referral.util.BranchCrossPlatformId.BranchCrossPlatformIdListener}
+     *                to callback with cross platform ids
+     *
+     */
     public void getCrossPlatformIds(BranchCrossPlatformIdListener callback) {
         new BranchCrossPlatformId(callback, context_);
     }
 
+    /**
+     * Gets the available last attributed touch data. Defaults to a 30 day attribution window, and sends -1 to request the default from the server.
+     *
+     * @param callback An instance of {@link io.branch.referral.ServerRequestGetLATD.BranchLastAttributedTouchDataListener}
+     *                 to callback with last attributed touch data
+     *
+     */
+    public void getLastAttributedTouchData(BranchLastAttributedTouchDataListener callback) {
+        prefHelper_.setLATDAttributionWindow(-1);
```

Comment:
What does `-1` mean?   Can this be a constant?   I see the comment about it sending "the default from the server", but that still should be a constant somewhere.